package.json

```
@@ -15,10 +15,9 @@
   "license": "ISC",
   "dependencies": {
     "@apollo/server": "4.11.0",
+    "@prisma/client": "^5.20.0",
```

Geralmente setamos a versão específica das libs externas para que se algo deixe de funcionar em futuras versões isso não afete nosso projeto.
```suggestion
    "@prisma/client": "5.20.0",
```

---

Olá, Samuel! Obrigada pelo feedback! Fiz as alterações sugeridas no package.json para definir as versões específicas.